.github/workflows/dispatch.yaml

```diff
@@ -12,63 +12,8 @@ env:
   IMAGE: ghcr.io/${{ github.repository }}/syfobehandlendeenhet:${{ github.event.inputs.gitCommit }}
 
 jobs:
-  build:
```

Comment:
Skal vi ikke bygge mer?

Reply:
Nei, siden vi uansett bygger og `check if` sjekken ikke funker med `ghcr`, så vil vi ikke bygge når vi også deployer. Det betyr at vi må vente på at koden bygger i PRen før vi deployer, men det gjør vi kanskje uansett 🤷🏼‍♂️ Deploy tar lenger tid når vi må bygge to ganger. Jeg, JML og Geir snakket om dette, har begynt å gjøre det i noen apper allerede: https://github.com/navikt/isoppfolgingstilfelle/pull/102

Jeg skal uansett rydde opp commitene her litt, så tenkte å trekke det ut i en egen PR å fjerne bygg-steget. Men foreløpig må jeg teste alt i dev og da sparer det meg litt tid 😊